src/Engine/Audio/SteamAudio.cpp

```diff
@@ -1,59 +1,47 @@
 #include "SteamAudio.hpp"
+#include <assert.h>
 
 #ifdef USINGMEMTRACK
 #include <MemTrackInclude.hpp>
 #endif
 
 SteamAudio::SteamAudio() {
-    context = nullptr;
-    environmentalRenderer = nullptr;
-}
-
-SteamAudio::SteamAudio(IPLContext * context, IPLhandle * environment) {
-    this->context = context;
-    this->environmentalRenderer = environment;
+    playerPos = IPLVector3{ 0,0,0 };
+    playerDir = IPLVector3{ 1,0,0 };
+    playerUp = IPLVector3{ 0,1,0 };
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)